scripts/variantstore/wdl/GvsUnified.wdl

```diff
@@ -0,0 +1,158 @@
+version 1.0
+
+import "GvsAssignIds.wdl" as AssignIds
+import "GvsImportGenomes.wdl" as ImportGenomes
+import "GvsCreateAltAllele.wdl" as CreateAltAllele
+import "GvsCreateFilterSet.wdl" as CreateFilterSet
+import "GvsPrepareRangesCallset.wdl" as PrepareRangesCallset
+import "GvsExtractCallset.wdl" as ExtractCallset
+
+workflow GvsUnified {
+    input {
+        # Begin GvsAssignIds
+        String dataset_name
+        String project_id
+
+        Array[String] external_sample_names
+        Boolean samples_are_controls = false
+
+        File? gatk_override
+        String? service_account_json_path
+        # End GvsAssignIds
+
+        # Begin GvsImportGenomes
+        Array[File] input_vcfs
+        Array[File] input_vcf_indexes
+        File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
+
+        Int? load_data_preemptible_override
+        # End GvsImportGenomes
+
+        # Begin GvsCreateFilterSet
+        String filter_set_name
+        Array[String] indel_recalibration_annotation_values = ["AS_FS", "AS_ReadPosRankSum", "AS_MQRankSum", "AS_QD", "AS_SOR"]
+        Int create_filter_set_scatter_count
+        Array[String] snp_recalibration_annotation_values = ["AS_QD", "AS_MQRankSum", "AS_ReadPosRankSum", "AS_FS", "AS_MQ", "AS_SOR"]
+
+        Int? INDEL_VQSR_max_gaussians_override = 4
+        Int? INDEL_VQSR_mem_gb_override
+        Int? SNP_VQSR_max_gaussians_override = 6
+        Int? SNP_VQSR_mem_gb_override
+        # End GvsCreateFilterSet
+
+        # Begin GvsPrepareRangesCallset
+        String extract_table_prefix
+
+        String query_project = project_id
```

Comment:
Duplicate with that on line 56.
And why is this even needed?

Reply:
Good catch! I'm surprised WDL allows completely duplicate declarations like that 😦 

Those other parameters are used in `GvsPrepareCallset` and `GvsExtractCallset`, though I do not know the "why" as to when they would be used.